### DISCUSSION TITLE
Remove a couple of points from dynamic width

### DISCUSSION THREAD

**Discussion by itaysharir** [FelixKratz/SketchyBar#247] [Q&A] [answered]
![Screen Shot 2022-09-02 at 23 05 28](https://user-images.githubusercontent.com/87126382/188228358-e46eb374-8cb9-4dd2-b6d8-8bcf06708f6b.jpg)

I want the width to still be dynamic, but get rid of the extra background on the sides

Thanks in advance!

**Comment by FelixKratz** [comment, marked as answer]
The background width at the sides is controlled by the left padding of the item and the right padding of the label, if both are drawing, as in the visualization shown on the website: https://felixkratz.github.io/SketchyBar/config/items. If either icon or label are not drawing, the excess width is controlled by the paddings of the remaining text.

Try this:
```bash
sketchybar --set <name> icon.padding_left=0 label.padding_right=0
```
and play with the values till you arrive at something you like. You could also go negative with these values.

**Comment by itaysharir** [comment]
Thanks!
